test/fixtures/mock-release.js

```diff
@@ -46,7 +46,6 @@ const setup = t => {
     nock.recorder.rec({ output_objects: true, dont_print: true })
   } else {
     nock.define(nock.loadDefs(fixturePath))
-    nock.disableNetConnect()
```

Comment:
This is an important safeguard.

Reply:
this was me figuring out I needed to generate fixtures :)